modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -2841,10 +2827,8 @@ the destination image will have the given size therefore the area of the boundin
 \par Reverse mapping
 
 You can get reverse mapping adding #WARP_INVERSE_MAP to `flags`
-\snippet polar_transforms.cpp InverseMap
```

Comment:
These snippets illustrated statement before them, without snippets meaning is lost. These two paragraphs must be rewritten, maybe similar snippets could be used from other sample.

Reply:
@mshabunin I have added the code block again, without referencing to the sample